packages/grid/x-grid-modules/src/components/styled-wrappers/getStyles.ts

```diff
@@ -0,0 +1,235 @@
+import { darken, fade, lighten, makeStyles } from '@material-ui/core/styles';
+
+export const getStyles = makeStyles((theme) => ({
+  'MuiDataGrid-root': {
+    lineHeight: theme.typography.pxToRem(24),
+    'box-sizing': 'border-box',
+    position: 'relative',
+    'font-family': theme.typography.fontFamily,
+    'letter-spacing': theme.typography.body2,
+    border: '1px solid #bdc3c7',
+    'border-radius': theme.shape.borderRadius,
+    outline: 'none',
+    display: 'flex',
+    flex: 1,
+    'flex-direction': 'column',
+
+    '& *': {
+      'box-sizing': 'border-box',
+    },
+
+    '& .main-grid-container': {
+      position: 'relative',
+      'flex-grow': 1,
+      'flex-direction': 'column',
+      display: 'flex',
+    },
+
+    '& .watermark': {
+      position: 'absolute',
+      'pointer-events': 'none',
+      color: '#8282829e',
```

Comment:
A note for self, I need to check that it contrasts enough on light and dark background.